src/main/java/com/google/api/codegen/viewmodel/ImportSectionView.java

```diff
@@ -17,15 +17,25 @@
 import com.google.auto.value.AutoValue;
 import com.google.common.collect.ImmutableList;
 import java.util.List;
+import javax.annotation.Nullable;
 
 @AutoValue
 public abstract class ImportSectionView {
   public abstract List<ImportFileView> standardImports();
 
   public abstract List<ImportFileView> externalImports();
 
+  /** This is a superset of the union of localImports() and sharedImports(). */
   public abstract List<ImportFileView> appImports();
 
+  /** Used in Python. */
```

Comment:
Can you make this documentation more descriptive.

Reply:
Done.